receiver/mongodbatlasreceiver/factory.go

```diff
@@ -82,5 +109,9 @@ func createDefaultConfig() config.Receiver {
 		Alerts: AlertConfig{
 			Enabled: defaultAlertsEnabled,
 		},
+		Logs: LogConfig{
+			Enabled:  true,
```

Comment:
Let's follow the pattern established by alerts and use a `defaultLogsEnabled` value.